Vagrantfile

```diff
@@ -131,3 +131,27 @@ Vagrant.configure("2") do |config|
     }
   end
 end
+
+## VM for proxy server
+Vagrant.configure("2") do |config|
+  config.vm.hostname = 'proxy-server.subman.example.com'
+  config.vm.box = 'centos/7'
+  config.vm.define 'proxy-server', autostart: false
+
```

Comment:
Created VM has two NIC (one of them uses private network). It should have only one NIC using public network. Look at definition of VMs in #1786 and obsolete comments from Kevin in this PR.